app/ui/browser/reducers/main-reducers.js

```diff
@@ -37,15 +37,26 @@ function getPageIndexById(state, id) {
 export default function basic(state = initialState, action) {
   switch (action.type) {
     case types.CREATE_TAB:
```

Comment:
For posterity, there was a related short convo in Slack. 

My concern is that we have two action pathways here for the same thing:

- `type.CREATE_TAB` creates the tab locally and then pushes the data to the backend through instrumentation
- `type.IPC_COMMAND_CREATE_TAB` is the result of firing `new-tab` on the backend and then creating the tab locally

We discussed if it would be too expensive to go over IPC for all "new tab" events. This would mean we'd no longer have the local version (`types.CREATE_TAB`). All new tabs would be created by firing off a `new-tab` event.

We aren't sure whether it would be too expensive performance-wise to do this. It might be worth it for the simplification of only having one path. However, there were concerns.

Since there isn't consensus around using `new-tab` for all new tabs, we can go ahead with having 2 different actions.